db/migrate/20241002185332_add_context_window_size_to_model_config.rb

```diff
@@ -0,0 +1,5 @@
+class AddContextWindowSizeToModelConfig < ActiveRecord::Migration[7.1]
+  def change
+    add_column :model_configs, :context_window_size, :integer
```

Comment:
What do you think about adding `_bytes` to this so the unit is clear?

Reply:
Hm, there is one other column, on ChunkingProfile, that is `size` without `_bytes`. Seems okay.